iocBoot/common/mps_link_node_change_units.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
+import os
+import sys
+import argparse
+from epics import caget, caput
+
+def change_units(PV, newHigh, newEGU, integrator=False):
```

Comment:
You also need to specify the `EGUL` field. In our examples, it is always 0, but that might not be always the case. Maybe you can add an argument for it with a default value = 0.

Reply:
I added a min value and support for changing EGUL into the script.